doc/MANUAL.adoc

```diff
@@ -309,12 +309,16 @@ this:
 2. Otherwise, if the environment variable `CCACHE_DIR` is set then use
    `$CCACHE_DIR/ccache.conf`.
 3. Otherwise, if <<config_cache_dir,*cache_dir*>> is set in the secondary
-   (system-wide) configuration file then use `<cache_dir>/ccache.conf`.
+   (system-wide) configuration file then use `<cache_dir>/ccache.conf`. The
+   system-wide configuration on Windows is `C:\ProgramData\ccache\ccache.conf`
```

Comment:
I don't see in the patch where `C:\ProgramData` is reffered. The part in the CMake file doesn't count.

Reply:
See above.

Reply:
This value is hardcoded, use `%allusersprofile%` instead.